xrpl/models/transactions/account_set.py

```diff
@@ -90,28 +92,57 @@ class AccountSetFlag(int, Enum):
     """Allow another account to mint and burn tokens on behalf of this account."""
 
 
-class AccountSetFlagInterface(FlagInterface):
+class AccountSetTfFlags(int, Enum):
     """
-    There are several options which can be either enabled or disabled for an account.
-    Account options are represented by different types of flags depending on the
-    situation. The AccountSet transaction type has several "AccountSet Flags" (prefixed
-    `asf`) that can enable an option when passed as the SetFlag parameter, or disable
-    an option when passed as the ClearFlag parameter. This TypedDict represents those
-    options.
+    Enum for AccountSet Transaction Flags.
 
-    `See AccountSet Flags <https://xrpl.org/accountset.html#accountset-flags>`_
+    Transactions of the TrustSet type support additional values in the Flags field.
+    This enum represents those options.
+    """
+
+    TF_REQUIRE_DEST_TAG = 0x00010000
+    """
+    The same as SetFlag: asfRequireDest.
+    """
+
+    TF_OPTIONAL_DEST_TAG = 0x00020000
+    """
+    The same as ClearFlag: asfRequireDest.
+    """
+
+    TF_REQUIRE_AUTH = 0x00040000
+    """
+    The same as SetFlag: asfRequireAuth.
+    """
+
+    TF_OPTIONAL_AUTH = 0x00080000
+    """
+    The same as ClearFlag: asfRequireAuth.
+    """
+
+    TF_DISALLOW_XRP = 0x00100000
+    """
+    The same as SetFlag: asfDisallowXRP.
+    """
+
+    TF_ALLOW_XRP = 0x00200000
+    """
+    The same as ClearFlag: asfDisallowXRP.
+    """
+
+
+class AccountSetFlagInterface(FlagInterface):
```

Comment:
Instead of replacing the `AccountSetFlagInterface` with tf flags, I believe we should have two objects. 

`AccountSetFlagAsfInterface` and `AccountSetFlagTfInterface` (open to other names as I know the acronym breaks down literally into `AccountSetFlagAccountSetFlagInterface` but the `asf` is distinctive so worth including in my opinion. 

They should both then allow for easy translation between the boolean `Interface` version and the `Flags` versions. 

I don't think we should have a generic `AccountSetFlagInterface` as I think that would be confusing. i'd prefer people explicitly choose `asf` or `tf`

Reply:
Oh sorry, I see the above conversation about naming. I agree with the defaults above.